src/js/Events.js

```diff
@@ -68,9 +68,10 @@ var Events = {
   SELECTION_CREATED: 'SELECTION_CREATED',
   SELECTION_MOVE_REQUEST: 'SELECTION_MOVE_REQUEST',
   SELECTION_DISMISSED: 'SELECTION_DISMISSED',
-  SELECTION_COPY: 'SELECTION_COPY',
-  SELECTION_CUT: 'SELECTION_CUT',
-  SELECTION_PASTE: 'SELECTION_PASTE',
+
+  CLIPBOARD_COPY: 'CLIPBOARD_COPY',
+  CLIPBOARD_CUT: 'CLIPBOARD_CUT',
+  CLIPBOARD_PASTE: 'CLIPBOARD_PASTE',
```

Comment:
@caleybrock Didn't you customize some copy-paste behavior in Piskel?  Since these event names changed, you may want to verify that our customized behavior is still present.

Reply:
I do believe all the behaviors have been updated with the new names, but it wouldn't hurt to have Caley check it out. :)

Reply:
yep, we updated and verified copy and paste when we did the merge.